src/ai/backend/agent/agent.py

```diff
@@ -112,7 +112,7 @@
 
 from . import __version__ as VERSION
 from . import resources as resources_mod
-from .exception import AgentError, ResourceError
+from .exception import AgentError, DockerContainerCreationError, ResourceError
```

Comment:
`DockerContainerCreationError`, name of the error, is too specific for docker backend. How about changing it to `ContainerCreationError`, without Docker?
```suggestion
from .exception import AgentError, ContainerCreationError, ResourceError
```

Reply:
I renamed it. Thanks!